server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -607,7 +607,7 @@ private static void initZooKeeper(Opts opts, String uuid, String instanceNamePat
       zoo.recursiveDelete(instanceNamePath, NodeMissingPolicy.SKIP);
     zoo.putPersistentData(instanceNamePath, uuid.getBytes(UTF_8), NodeExistsPolicy.FAIL);
 
-    final byte[] EMPTY_BYTE_ARRAY = new byte[0], ZERO_CHAR_ARRAY = new byte[] {'0'};
+    final byte[] EMPTY_BYTE_ARRAY = new byte[0], ZERO_CHAR_ARRAY = {'0'};
```

Comment:
Variables should really have their own declarations with their own modifiers. I wonder if we can enforce this with checkstyle.

Reply:
fixed in 96d0411ccda8fa3